test/README.md

```diff
@@ -10,7 +10,66 @@ eval $(minishift docker-env)
 make test-e2e-local
 ```
 
-## Steps to verify operator registry
+## Steps to verify operator registry on OCP4
```

Comment:
There is not much difference with the existing document other than a couple of namespaces.
https://github.com/redhat-developer/devconsole-operator/tree/master/test#steps-to-verify-operator-registry